addons/account/report/account_invoice_report.py

```diff
@@ -132,37 +132,3 @@ def _where(self):
                 AND line.account_id IS NOT NULL
                 AND line.display_type = 'product'
         '''
-
-
-class ReportInvoiceWithoutPayment(models.AbstractModel):
-    _name = 'report.account.report_invoice'
-    _description = 'Account report without payment lines'
-
-    @api.model
-    def _get_report_values(self, docids, data=None):
-        docs = self.env['account.move'].browse(docids)
-
-        qr_code_urls = {}
-        for invoice in docs:
-            if invoice.display_qr_code:
-                new_code_url = invoice._generate_qr_code(silent_errors=data['report_type'] == 'html')
```

Comment:
You sure this is a good idea ? If I remember well, the reason it was done this way was that, in case there was an error (if the QR-method is eligible but not all required fields are set, so it's a common one), we would raise before being in the Qweb rendering. And that's important, because raising an error in Qweb triggered a huge and unreadable traceback, while here it only consists of an error message to the user.